packages/deployment/app/cloudformation.template.yml

```diff
@@ -450,6 +451,12 @@ Resources:
             Ref: {{../es.name}}DomainEndpoint
         {{/if}}
         {{/if}}
+        {{#ifNotEquals this.useMessageAdapter "true"}}
+        {{#if this.envs.CUMULUS_MESSAGE_ADAPTER_DIR}}
+        {{else}}
```

Comment:
Should there be a `CUMULUS_MESSAGE_ADAPTER_DIR` env var specified here using `this.envs.CUMULUS_MESSAGE_ADAPTER_DIR`?

Reply:
The purpose of this is to allow users to override the overall setting.   We're setting /opt for *all* lambdas unless it's defined for that particular lambda or the legacy flag is enabled.  We have some users that are currently packaging the cumulus-message-adapter in their script. 

Typing the above leads me to the conclusion that this is probably not adequately documented.  I'll push something up with a better explanation of the configuration.

Reply:
@markdboyd  how does 71a87a38e read to you?

Reply:
What is this referring to?

Reply:
71a87a38e

Reply:
Apparently it won't render later  if the push didn't succeed before you add the comment.